src/components/common/buttons/ClaimButton.js

```diff
@@ -1,14 +1,40 @@
 // @flow
-import React from 'react'
-import { Platform } from 'react-native'
+import React, { useCallback } from 'react'
+import { Animated, Platform, View } from 'react-native'
 import { PushButton } from '../../appNavigation/PushButton'
 import { withStyles } from '../../../lib/styles'
 
-const ClaimButton = ({ screenProps, styles }) => (
-  <PushButton routeName="Claim" testID="claim_button" screenProps={screenProps} style={styles.claimButton}>
-    Claim
-  </PushButton>
-)
+const ClaimButton = ({ screenProps, styles, animated }) => {
+  const [containerSize, setContainerSize] = React.useState([])
```

Comment:
Default value should be an object

Reply:
Also the name of the state does not make any sense. Should be something like pushButtonTranslate instead